### DISCUSSION TITLE
keep-alive

### DISCUSSION THREAD

**Discussion by tephyrnex** [UweKeim/ZetaIpc#16] [General]
What is the thinking behind disabling keep-alive connections _(ie. MyWebClient & MyModule)_?  Does that cause some specific problem with Zeta-IPC?

**Comment by UweKeim** [comment]
As you can see in the source code's comment, I got the idea from [this Stack Overflow question and answers](https://stackoverflow.com/q/896207/107625).

I'm not sure anymore on why exactly I did it. Possibly because I wanted it really stateless and without possible caching. Most likely this was one way to achieve this.

[See also `Keep-Alive` documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Keep-Alive).
